src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpWhosInControlApplicationControllerTests.cs

```diff
@@ -511,6 +523,66 @@ public void Confirm_trustees_presents_list_of_trustees()
             trusteesCount.Should().Be(2);
         }
 
+
+        [Test]
+        public void Confirm_trustees_exempted_returns_to_tasklist()
```

Comment:
Test is not named to show its purpose.... `Confirm_NoTrusteesAndProviderExempted_ReturnsToTaskList`

Reply:
ok